lab-goat/lib/read-write-bitmap.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const Bitmap = require ('../model/bitmap');
+const fs = require('fs');
+
+
+let originalObj = module.exports = {};
+
+originalObj.greyOut = (callback) => {
+
+  fs.readFile(`${__dirname}/../../assets/palette-bitmap.bmp`, (err, data) => {
+    if (err) console.error(err);
+    let bmp = new Bitmap(data);
+
+    // console.log('Here is the constructor bmp:', bmp);
+    // console.log('Here is the bmp constructor property pixelArray:', bmp.pixelArray);
+
+    let x = bmp.pixelArray;
+
+    // console.log('This is the pixelArray', x);
+
+    for(let i = 0; i < x.length; i += 4) {
```

Comment:
Would have liked to see this further modularized, but getting the primary functionality in place was a great first step!